lab_1_keywords_tfidf/start.py

```diff
@@ -31,6 +36,18 @@
     with open(CORPUS_FREQ_PATH, 'r', encoding='utf-8') as file:
         corpus_freqs = json.load(file)
 
-    RESULT = None
-    # DO NOT REMOVE NEXT LINE - KEEP IT INTENTIONALLY LAST
-    assert RESULT, 'Keywords are not extracted'
+    # RESULT = None
+    # # DO NOT REMOVE NEXT LINE - KEEP IT INTENTIONALLY LAST
+    # assert RESULT, 'Keywords are not extracted'
```

Comment:
Do not forget not to remove those lines and to keep them intentionally concluding the whole file